design/one-pager-oam-workflow.md

```diff
@@ -0,0 +1,227 @@
+# OAM POC Workflow
+* Owner: Dan Mangum (@hasheddan)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+This document describes a basic workflow for utilizing OAM types as implemented
+in [1276](https://github.com/crossplane/crossplane/pull/1276). The controllers described are for an initial POC and will be moved out of core
+Crossplane in later iterations.
+
+1. Crossplane Installation
```

Comment:
Consider changing its format to Header 2?

Reply:
Like
```suggestion
## 1. Crossplane Installation
```

Reply:
Will do! Thanks for suggestion :)